packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 * Go To Definition and Find All References for types used with instanceOf ([PR #376](https://github.com/forcedotcom/salesforcedx-vscode/pull/376))
 * Go To Definition for implicit constructors across Apex classes ([PR #376](https://github.com/forcedotcom/salesforcedx-vscode/pull/376))
 
+### Fixed
+#### salesforcedx-vscode-core
+
+* Support invoking the new Salesforce CLI which were installed using the new installers on Windows ([PR #386](https://github.com/forcedotcom/salesforcedx-vscode/pull/386)
```

Comment:
The new Salesforce CLI what? Would this be accurate? 

* Support invoking Salesforce CLI commands for CLI installations that used the new installers on Windows ([PR #386](https://github.com/forcedotcom/salesforcedx-vscode/pull/386)

Reply:
Also, we're missing an ending parenthesis.